### PR TITLE
fix: build error on android

### PR DESCRIPTION
**Pull request by alantoa**
# Why 
fix some errors on Android: 
- got this error when I build the example project on Android because [react-native-ios-context-menu](https://github.com/dominicstop/react-native-ios-context-menu) uses the default [android/gradle.properties file](https://github.com/dominicstop/react-native-ios-context-menu/blob/master/android/build.gradle), so we need to override it. 

 

- `SubContent` not working on Android, because it uses old `Content` to pick children.

# How 
- use [expo-build-properties ](https://docs.expo.dev/versions/latest/sdk/build-properties/) plugin to fix it. 
- use `SubContext` to replace the `pickChildren` method params with `Context` on Android. 

# Test Plan 

- remove the `expo/android` folder and then run `expo run:android -d Pixel_5_API_33` or other devices.

# Preview 

## Before 
image

## After 

image

### CODE REVIEW

**Comment by nandorojo** [comment]
Interesting…should we send an issue + PR to react native ios context menu then?

**Comment by nandorojo** [comment]
Also, naive question, but does removing this dependency on Android inside of `react-native.config.js` not fix this?

**Comment by alantoa** [comment]
hey @nandorojo, thanks for the comments. 

use `react-native.config.js` absolutely can fix the `react-native-ios-context-menu` issue on Android, but I just noticed the `@react-native-menu/menu` also has a build issue, it's related to the Gradle version, so I created a [PR](https://github.com/react-native-menu/menu/pull/474) for fix this. 

so I think using the [expo-build-properties](https://docs.expo.dev/versions/latest/sdk/build-properties/) plugin is necessary for now utils the PR merged.

**Comment by alantoa** [comment]
hey @nandorojo, `@react-native-menu/menu` has merged and released my [PR](https://github.com/react-native-menu/menu/pull/474),  so I just:
- removed [expo-build-properties](https://docs.expo.dev/versions/latest/sdk/build-properties/)
- upgraded `@react-native-menu/menu`  version to `v0.7.2`
- add `react-native-ios-context-menu` to react-native config 

now is pretty working!

**Comment by nandorojo** [comment]
@alantoa is any change required for Zeego here?

**Comment by alantoa** [comment]
hey, @nandorojo no, this PR just includes bug fixes on Android.

**Comment by fobos531** [comment]
@nandorojo The android build error part of the issue might be fixed by this https://github.com/dominicstop/react-native-ios-context-menu/pull/59#issuecomment-1405133828

**Comment by alantoa** [comment]
@nandorojo could we merge this PR? it's safe just fixed some android some issue, I'm already using this patch for a long time. 
let me know if need I to do any updates! thanks!

File: packages/zeego/src/menu/create-android-menu/index.android.tsx [hunk 1]
```diff
@@ -359,7 +359,7 @@ If you want to use a custom component as your <Content />, you can use the creat
             if (triggerItem) {
               const nestedContent = pickChildren<MenuContentProps>(
                 child.props.children,
-                Content
```
Comment (alantoa) [inline packages/zeego/src/menu/create-android-menu/index.android.tsx:362]: accurately the `Content` is incorrect here, it should be `SubContent`.

**Pull request merged**